bluemira/equilibria/equilibrium.py

```diff
@@ -1073,13 +1073,8 @@ def set_vcontrol(self, vcontrol_str: Optional[str] = None):
             vcontrol_type = VerticalPositionControlType(vcontrol_str)
             if vcontrol_type is VerticalPositionControlType.VIRTUAL:
                 self.controller = VirtualController(self, gz=2.2)
-            elif vcontrol_type is VerticalPositionControlType.FEEDBACK:
+            if vcontrol_type is VerticalPositionControlType.FEEDBACK:
```

Comment:
I think probably more robust is the below as we haven't dealt with any other control type
```suggestion
            else:
```